### PR TITLE
update documentation method for helpers

### PR DESCRIPTION
**Pull request by j042**
# Overview
Instead of grabbing onto `__docs__`, this PR uses built-in functools to document helper functions.

## Addressed issues
*  aesthetics 

## Checklists
* [ ] Documentations are up-to-date.

### CODE REVIEW

File: splinepy/helpme/extract.py [hunk 1]
```diff
@@ -410,7 +412,7 @@ def spline(spline, para_dim, split_plane):
     spline: Spline / Multipatch
       (`self`-argument if called via extract member of a spline)
     para_dim : int
-      parametric dimension to be extract ted
+      parametric dimension to be extracted
```
Comment (j042) [inline splinepy/helpme/extract.py:415]: @jzwar Theodore?

**Review by jzwar — APPROVED** [review]
Love it

File: splinepy/helpme/extract.py (hunk 1 above)
Comment (j042) [reply splinepy/helpme/extract.py:415]: is our spline extractor

File: splinepy/helpme/extract.py [hunk 2]
```diff
@@ -681,12 +692,13 @@ def spline(self, splitting_plane=None, interval=None):
 
         Parameters
         ----------
-        splitting_plane : int / dictionary (int : (float))
-          if integer : parametric dimension to be extracted
-          if dictionary : list of splitting planes and ranges to be passed
-        interval : float / tuple<float,float>
+        splitting_plane : int or dictionary {int : (float, float))
```
Comment (clemens-fricke) [inline splinepy/helpme/extract.py:695]: Was it your intend to change the first brace? If it was you should match also the last brace.
```suggestion
        splitting_plane : int or dictionary {int : (float, float)}
```

**Review by clemens-fricke — APPROVED** [review]
LGTM. Thanks

File: splinepy/helpme/extract.py (hunk 2 above)
Comment (j042) [reply splinepy/helpme/extract.py:695]: nice catch

**Pull request merged**